README.md

```diff
@@ -2,7 +2,251 @@
 |-----------|:----------|
 
 # StandardKit Styles
-### SCSS Framework for the StandardKit Design & Component System
+
+SCSS Framework for the StandardKit Design & Component System
+
+## Principles
+
+### Standards & Defaults
+### Multi-Path Solutions
+### Cascading Configuration
+### Built-to-Extend
+
+
+## Modifiers, Flavors, Components & Types
+
+
+### Modifiers
+A modifier is a characteristic that can be modified. 
+Something that you can change.
+Examples are:
+- `color`
+- `fill`
+- `margin`
+
+<sub>Example selection, please check the documentation website for a full list.</sub>
+
+All modifiers work the same from the outside. For example to apply a different `padding` to a component, you would say:
+```html
+<div class="padding--xl">
+    A div with padding 'xl'
+</div>
+```
+
+As you can probably tell, we use two hyphens (BEM-style) to select an alternative.
+
+
+### Flavors
+Modifiers can have a number of flavors defined. For example:
+```scss
+$color: (
+  error: red,
+  warning: orange,
+  info: blue,
+  success: green
+);
+```
+<sub>This is a simplified example. Check the `theme` for the actual values.</sub>
+
+
+### Custom Flavors
+There are two types of modifiers; `static` & `configurable`. 
+
+#### Static Modifiers
+An example of a `static` modifier is: 
+- `case` (`lowercase`, `uppercase` `capitalized`).
+
+Since there is a limited / hardcoded number of options here, it doesn't make sense to expand. 
+
+<sub>
+If however you feel that there is a usecase to expand on one of the static modifiers, 
+please create an issue and let us know! <br>
+StandardKit is never perfect and you should be able to use it the way you want it.
+</sub>
+
+#### Configurable Modifiers
+Most modifiers are configurable. This means that you can add your own flavors, as well as override pre-defined ones.
+For example:
+```scss
+// Override the error color to 'purple' and adding our own 'custom' color
+$colors: (
+  error: purple,
+  custom: hotpink
+);
+```
+Now you could use this the following:
```

Comment:
Typo. Maybe "Now you could use this the following way:" or "Now you could use it as follows:"?